pallets/vc-management/src/lib.rs

```diff
@@ -129,10 +129,12 @@ pub mod pallet {
 		},
 		// event that should be triggered by TEECallOrigin
 		// a VC is just issued
+		// we have `id_graph_hash` field since vc request could create the IDGraph
 		VCIssued {
 			identity: Identity,
 			assertion: Assertion,
 			index: VCIndex,
+			id_graph_hash: H256,
```

Comment:
Why do we add this value here ? Just to log that idgraph was created or because we assume client will use it ? If the latter I think client will need to get idgraph anyway.

Reply:
it brings consistency to other operations that mutate the id_graph too. I think it is nice to have. get_id_graph from client sounds tempting until one remembers that the user has to sign that request every time.

Reply:
Just for logging purpose - we don't expect the client to use this event